contentpool_client.info.yml

```diff
@@ -1,5 +1,9 @@
-name: 'Contentpool client'
+name: 'Contentpool Client'
 type: module
-description: 'Connects the site to a contentpool.'
+description: 'Connects the site to a remote contentpool.'
 package: Contentpool
 core: 8.x
+
+dependencies:
+  - relaxed
```

Comment:
are those all dependencies needed? Multiversion?

The dependency format should use project:module now, please check and adapt.

Reply:
We currently rely on the default replication-module mechanism (workspace to workspace) which provides us with much in terms of normalization but also introduces multiversion. If we want to skip the replication-stack on the client we'd have to roll many features of the replication-stack in a "one-way" architecture.